packages/Liquid-UI.package/LQPollDraftBuilder.class/instance/getDuplicates..st

```diff
@@ -0,0 +1,8 @@
+duplicateDetection
+getDuplicates: aCollection
```

Comment:
This method does not belong the UI directly. Such logic should be located in a helper class (utility-approach). This makes testing much simpler and decouples the UI. Have a look at the LQChoiceProcessor.